vpc/service/branch_enis.go

```diff
@@ -1404,3 +1406,45 @@ func (vpcService *vpcService) createBranchENI(ctx context.Context, tx *sql.Tx, s
 
 	return iface, nil
 }
+
+func (vpcService *vpcService) ListTrunkToBranchENIMappings(ctx context.Context, req *vpcapi.ListTrunkToBranchENIMappingsRequest) (*vpcapi.ListTrunkToBranchENIMappingsResponse, error) {
+	ctx, span := trace.StartSpan(ctx, "listTrunkToBranchENIMappings")
+	defer span.End()
+
+	mappings := []*vpcapi.TrunkENIMapping{}
+
+	tx, err := vpcService.db.BeginTx(ctx, &sql.TxOptions{
+		ReadOnly: true,
+	})
+	if err != nil {
+		err = errors.Wrap(err, "Cannot start database transaction")
+		tracehelpers.SetStatus(err, span)
+		return nil, err
+	}
+	rows, err := tx.QueryContext(ctx, `
+	SELECT
+	branch_eni,
+	trunk_eni
+	FROM assignments
+	JOIN branch_eni_attachments ON assignments.branch_eni_association = branch_eni_attachments.association_id
+`)
+	for rows.Next() {
+		var branchENIId, trunkENIId string
+		err = rows.Scan(&branchENIId, &trunkENIId)
+		if err != nil {
+			err = errors.Wrap(err, "Error scanning for all trunk to branch ENI mappings")
+			tracehelpers.SetStatus(err, span)
+			return nil, err
+		}
+		logger.G(ctx).Debug(fmt.Sprintf("Returning trunk to branch eni map: %v => %v", trunkENIId, branchENIId))
```

Comment:
You should use with field, a la:
```
logger.G(ctx).WithField("trunkENI", trunkENIId).Debug("Returning trunk to branch ENI map")
```
(I think that this may be a little too verbose).

Reply:
It still ends up on one line, I've added `WithFields` so that it prints both